extern/algebra-plugins/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building Algebra Plugins as part of the Detray project" )
+
+# Declare where to get Algebra Plugins from.
+FetchContent_Declare( AlgebraPlugins
+   GIT_REPOSITORY "https://github.com/krasznaa/algebra-plugins.git"
+   GIT_TAG "e6245fcab63a2f26581fabaf9a31f78a6dd7d3d3" )
+
+# Options used in the build of Algebra Plugins.
+set( ALGEBRA_PLUGIN_BUILD_TESTING FALSE CACHE BOOL
```

Comment:
See comment above: Can this be configured by detray flags or do agebra-plugins flags need to be set for a detray cmake call?